lib/xandra/cluster.ex

```diff
@@ -345,6 +345,15 @@ defmodule Xandra.Cluster do
       *Available since v0.18.0*.
       """
     ],
+    use_rpc_address_for_peer_address: [
```

Comment:
`peer` was the original implementation. I don't know why you would use `peer` instead of `rpc_address` and not much information when I google this, so going to phone a friend. @relistan  do you know or think this should just solely use `rpc_address` and no longer use `peer`? (I think that is what @whatyouhide is asking).

Reply:
Yes that's what I’m saying. @jacquelineIO I also recommend doing just what any other C* driver maintained by Datastax is doing. I always reference the [Python driver](https://github.com/datastax/python-driver). I don't have much time to look into this until maybe Sunday, so if you want you can just take a look there and use what they're using 😉

Reply:
Looks like it is using `rpc_address` with a fallback to `peer` if `rpc_address` isn't available

- https://github.com/datastax/python-driver/blob/8ba0a5ccd71b66c77ee58994ed9da9ea34ff4cbe/cassandra/cluster.py#L3782
- https://github.com/datastax/python-driver/blob/8ba0a5ccd71b66c77ee58994ed9da9ea34ff4cbe/cassandra/metadata.py#L3381


So I can remove this flag and have a preference for using `rpc_address`, which the code is basically already set up to do

Reply:
Yes, let's do that so that we don't have to put this decision on users 🙃 Thanks!

Reply:
Sorry, catching up here. That sounds right to me, then.

Reply:
@jacquelineIO any news on this and timeline on when I can expect these changes to go in? I'm planning the 0.18.0 release, that's why I ask 🙃